src/components/LoadingMessage/LoadingMessage.less

```diff
@@ -0,0 +1,33 @@
+.lucid-LoadingMessage {
+
+	.lucid-Icon {
+		display: inline-block;
+	}
+
+	@LoadingMessage-padding: @size-standard;
```

Comment:
`border-box` made this variable unnecessary, but we should still talk about this